src/ui/server/graphql/mutations/update-application/companyPostDataMigration.ts

```diff
@@ -0,0 +1,11 @@
+import gql from 'graphql-tag';
+
+const updateCompanyPostDataMigrationMutation = gql`
+  mutation updateCompanyPostDataMigration($id: String!, $company: CompanyInput!) {
+    updateCompanyPostDataMigration(id: $id, company: $company) {
+      success
```

Comment:
do you not need to also return  apiError, notFound?

Reply:
No, this is a mutation for updating company data in the DB (with data obtained from Companies House)

The Companies house search is already setup in a different query